code/menuui/mainhallmenu.cpp

```diff
@@ -2365,7 +2370,7 @@ void parse_one_main_hall(bool replace, int num_resolutions, int &hall_idx, int &
 		for (int idx = base_num; idx < m->num_misc_animations; idx++) {
 			// anim delay
 			required_string("+Misc anim delay:");
-			stuff_int(&m->misc_anim_delay.at(idx).at(0));
+			stuff_int(&m->misc_anim_initial_delay.at(idx));
```

Comment:
The original line should be kept somehow.  I suggest restoring the original line and then setting the initial delay equal to the first delay.

Reply:
This doesn't make much sense to me. The original behaviour was bogus; whatever value is set here is never going to be valid later on. Whatever value is put in here is going to be nuked anyway once we initialize the mainhall for proper use.

Reply:
It doesn't sit right with me to initialize indexes 1 and 2 but not 0.  It may not be needed with this fix, but if someone comes along and changes things later, setting the value would keep things consistent.

Reply:
If we have to set this value to something to preserve symmetry, we should set it to -1 and not something completely invalid.

Reply:
Setting it to -1 is ok with me.

Reply:
Done.